auth/loign.router.js

```diff
@@ -0,0 +1,40 @@
+const express = require('express');
+const utils = require('./utils');
+const Users = require('../models/Users.model');
+const { RouterFactory } = require('express-router-factory');
```

Comment:
Aren't you glad you took the time on the front end project to build some reusable elements?

Reply:
yes, very happy.
Did you noticed that I published it to npm? I use it as any other regular package dependency.
https://www.npmjs.com/package/express-router-factory

I still need to add a readme and instructions to the package in npm.